src/Compute/Compute/help/New-AzDiskConfig.md

```diff
@@ -25,8 +25,8 @@ New-AzDiskConfig [[-SkuName] <String>] [-Tier <String>] [-LogicalSectorSize <Int
  [-DiskEncryptionSetId <String>] [-EncryptionType <String>] [-DiskAccessId <String>]
```

Comment:
An example should be provided or edited on at least one of these .md files to show users how to use the new parameters. The script should be run locally first to ensure it actually works.